kolibri/core/device/kolibri_plugin.py

```diff
@@ -0,0 +1,22 @@
+from kolibri.core.auth.hooks import FacilityDataSyncHook
+from kolibri.plugins.hooks import register_hook
+
+
+@register_hook
+class SingleUserExamSyncHook(FacilityDataSyncHook):
```

Comment:
Shouldn't affect anything, but I think this name is copypasta.

Reply:
:yum: will fix.